content/en/developers/local-network.md

```diff
@@ -22,8 +22,8 @@ This process requires you to use multiple terminal windows, so you might find a
 
 Since spinning up a local-net requires the `lotus` daemon, you need to have Lotus installed. If you have already installed a Lotus node on this computer you can move onto the [next section ↓](#environment-setup)
 
-- [MacOS]({{< relref "macos" >}})
-- [Linux]({{< relref "ubuntu" >}})
+- [MacOS]({{< relref "../lotus/install/macos" >}})
+- [Linux]({{< relref "../lotus/install/linux" >}})
```

Comment:
The point of `relref` is that we can just use the title/identifier of the page and Hugo will figure out where it is. When you use relative links like this, Hugo will fail to build if we ever move the pages around. See the [Hugo docs here](https://gohugo.io/functions/relref/)

Reply:
We _should_ be able to add an `identifier` to pages that have conflicts. I'll have a play around with this now.

Reply:
Ack, let me know if we have a way to avoid this as I would prefer to use reflink as well. Much easier to move stuff around.

Reply:
I can't find a quick fix, so let's use your suggestion for now. I'll keep digging in the Hugo docs to see if there's a way around this relref issue.